.pre-commit-config.yaml

```diff
@@ -5,13 +5,10 @@ repos:
       - id: check-yaml
       - id: trailing-whitespace
       - id: check-toml
-  - repo: https://github.com/psf/black
-    rev: 23.7.0
+  - repo: https://github.com/psf/black-pre-commit-mirror
+    rev: 23.9.1
     hooks:
       - id: black
-        language_version: python3
-        args:
-          - "--preview"
```

Comment:
Why did you remove those?

Reply:
@KareemMAX Sorry, I removed by mistake. Now I have removed the changes made by black as you said and only updated the ` .pre-commit-config.yml` file, Thank you.